packages/nimble/inst/include/nimble/accessorClasses.h

```diff
@@ -161,7 +161,8 @@ class SingleVariableMapAccessBase {
  public:
   int offset, length;
   bool singleton;
-  vector<int> sizes, strides; 
+  vector<int> sizes, strides;
+ SingleVariableMapAccessBase() : offset(0), singleton(false) {length = 0;}; /* length(0) triggers the Rf_length macro mixup */
```

Comment:
Remove trailing semicolon here and everywhere else.

Also consider simplifying to `... : offset(0), length(0), singleton(false) {}`